apps/prairielearn/src/ee/institution/admin/saml.ts

```diff
@@ -1,25 +1,22 @@
-// @ts-check
-const { Router } = require('express');
-const asyncHandler = require('express-async-handler');
-const pem = require('pem');
+import { Router } from 'express';
+import asyncHandler = require('express-async-handler');
+import * as pem from 'pem';
 
-const error = require('@prairielearn/error');
-const sqldb = require('@prairielearn/postgres');
-const { InstitutionAdminSaml } = require('./saml.html');
-const {
+import * as error from '@prairielearn/error';
```

Comment:
Side comment, but this syntax seems to be hard for VSCode intellisense to automatically discover. Whenever I have some code that's doing `error.make()`, it can't figure out how to add the import line for `error` by itself. Could we change `@prairielearn/error` so that we can `import { error } from ...` or whatever VSCode is expecting?

Reply:
One step ahead of you:

https://github.com/PrairieLearn/PrairieLearn/blob/b56f252e3ade31c49f6a41c4a98e28b82b4f1588/packages/error/src/index.ts#L13-L14

This is more "canonical" that exporting an object with functions on it - just export the functions directly!

I think the _best_ thing to do here is to just transform these from functions to `Error` subclasses so we can just do something like this:

```ts
import { ErrorWithStatus } from '@prairielearn/error';

throw new ErrorWithStatus(404, 'thing was not found');
```

This is much more friendly to intellisense *and* it will show the actual location of the error in the stacktrace (as opposed to having `@prairielearn/lib/dist/index.js` as the first line in the stack trace).

All that said: these are changes for another day and another PR.